src/python/WMCore/WMSpec/Persistency.py

```diff
@@ -74,8 +77,16 @@ def load(self, filename):
             # use own request class so we get authentication if needed
             from WMCore.Services.Requests import Requests
             request = Requests(filename)
-            data = request.makeRequest('', incoming_headers={"Accept": "*/*"})
-            self.data = pickle.loads(data[0])
+            data = request.makeRequest('', incoming_headers={"Accept": "*/*"}, decoder=False)[0]
+            #if PY3:
```

Comment:
Self-reminder: This block can probably go away.

Reply:
Does `decoder=False` here solve your problem with py3 that you mention in the comment here below? otherwise, we can come up with a new argument for the `decoder=` that solves this

Reply:
Yes, I was fighting with encoding/decodings, but in the end that was all that I needed :-D